### PR TITLE
Unit tests for plan builders

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

File: api-model/src/main/java/io/enmasse/admin/model/v1/AddressSpacePlanSpec.java [hunk 1]
```diff
@@ -81,6 +81,6 @@ public void setAddressPlans(List<String> addressPlans) {
     }
 
     public List<String> getAddressPlans() {
-        return Collections.unmodifiableList(addressPlans);
+        return addressPlans;
```
Comment (lulf) [inline api-model/src/main/java/io/enmasse/admin/model/v1/AddressSpacePlanSpec.java]: This should not be modifyable

Comment (kornys) [reply api-model/src/main/java/io/enmasse/admin/model/v1/AddressSpacePlanSpec.java]: this cause NPE

Comment (kornys) [reply api-model/src/main/java/io/enmasse/admin/model/v1/AddressSpacePlanSpec.java]: @lulf that is the reson why I remove that

**Comment by lulf** [comment]
@enmasse-ci run tests testcase="plans.PlansTest"

**Comment by lulf** [comment]
@enmasse-ci run tests

**Pull request merged**